C/plugins/storage/postgres/connection.cpp

```diff
@@ -144,7 +144,7 @@ SQLBuffer	jsonConstraints;	// Extra constraints to add to where clause
 							{
 								sql.append((*itr)["column"].GetString());
 								sql.append(" AT TIME ZONE '");
-								sql.append((*itr)["timezone"]);
+								sql.append((*itr)["timezone"].GetString());
```

Comment:
We might check the provided timezone with IsString() in order to prevent corner case issues.